src/app/api/habitcontracts/create/route.js

```diff
@@ -42,12 +38,27 @@ export async function POST(req) {
     status: 'active',
     createdAt: Timestamp.now(),
     lastVerifiedDate: '',
+    habitTitle,
```

Comment:
why not just title subtitle?

Reply:
to be sure - can be removed in later PRs